include/ck/tensor_operation/gpu/device/impl/device_pool2d_fwd_impl.hpp

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <iostream>
+#include <sstream>
+
+#include "ck/tensor_description/tensor_descriptor.hpp"
+#include "ck/tensor_description/tensor_descriptor_helper.hpp"
+#include "ck/tensor_operation/gpu/device/reduction_operator_mapping.hpp"
```

Comment:
Remove this including since it is not used

Reply:
fdffa3a9da6ccade1a15d91fcae60197097792e8